TeaDiary/Models/DisplayTeaViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace TeaDiary.api.Models
+{
+    public class DisplayTeaViewModel
+    {
+        public int? Id { get; set; }
+
+        public DisplayTeaViewModel(int? id)
+        {
+            this.Id = id;
+        }
+
+        public bool Create
```

Comment:
`Create` is an action. The name of a boolean property needs to be an adjective or a participle. Think `IsNewTea`, or `isCreate`